src/Opserver.Web/Controllers/GraphController.Spark.cs

```diff
@@ -169,14 +172,14 @@ public ActionResult SQLCPUSpark(string node)
             return SparkSVG(points, 100, p => p.ProcessUtilization, start);
         }
 
-        public async Task<ActionResult> SparkSvgAll<T>(Func<Node, Task<List<T>>> getPoints, Func<Node, List<T>, long> getMax, Func<T, double> getVal) where T : IGraphPoint
```

Comment:
@NickCraver bug fix here: the "all" sparkline cache was being re-used for different time series so whichever one was fetched first would win! This commit adds a suffix so we can have a different cache for CPU, Network and Memory